### ISSUE TITLE
Selling items with Direct Transaction

### ISSUE THREAD

**Issue by YaibaToKen** [aura-project/aura#548]
So... the player seems to be frozen without being able to leave the shop when attempting to trade something directly to the bank, which is the default action when you sell something priced over 50k I believe :( this becomes a bother as the player is left without the possibility to sell said item...

`
[Unimplemented] - PacketHandlerManager: Handler for '715A', '?'.
[Debug] - Op: 0000715A ?, Id: 0010000000000002
001 [00500000000000FA] Long   : 22517998136852730
`

That's what the Channel Server logs, in case it's of any use.

**Comment by exectails** [comment]
Sounds like I might have missed an OP change after the MusicQ update.

**Comment by YaibaToKen** [comment]
Should I try and log a transaction with an item of similar value on officials? Or is there anything else I can get from the aura server to help with debug?

**Comment by exectails** [comment]
If my assumption is right we won't need a lot of debugging, it will be obvious from the OP codes of the packets we get.

**Comment by exectails** [comment]
Wait, *selling* with Direct Transaction? That's a thing? How does it work? o.o

**Comment by eotetb** [comment]
It is, for items that sell for over 50k gold, the direct transaction check box is automatically checked.

**Comment by exectails** [comment]
That probably won't be necessary, @eotetb, if this is just an alternate version of the sell packet, just for Direct Transaction. But if you're online anyway or something, feel free to log it anyway, just in case.

**Comment by exectails** [comment]
The "new" pack actually wasn't a new selling packet, but rather a check whether you actually can use direct transaction to sell that item. Adding that was pretty easy, but next up I had to add the actual direct transaction.

Problem here is that the fee is vastly different. It's not a static 5% like for buying. I came up with a formula, but it's not 100% accurate, so it might be off by 50 gold or something when selling a very expensive item. I'll leave that in for now, since I couldn't find the actual formula yet, and I doubt many are ever gonna notice that, but if it does become a problem somehow we can disable direct transaction selling until we find the actual formula.
